### DISCUSSION TITLE
Help using BDFR to scrape existing json

### DISCUSSION THREAD

**Discussion by ebrummer** [Serene-Arc/bulk-downloader-for-reddit#887] [General]
Hello,
I've already used Expanse (https://github.com/jc9108/eternity/) to scrape all of my saved posts/comments/my posts/my comments/my upvotes etc/import my reddit data export (including more then the 1000+ reddit limit for the regular feed.) I have a .json with texts of posts and links for them all.
Is there a way I can feed that json in to BDRF to scrape the links/related comments/post etc? I'm sure its possible if my python-fu existed.
That way I don't need to be limited by reddit's 1000 limitation. Or can I import my reddit exported file to scrape?

**Comment by Serene-Arc** [comment]
You'd have to take out the IDs in the JSON file and make it so that they're one ID per line in a text file. This is very easy to do in python, just a snippet of a few lines. I can write the snippet for you if you'd like. You just need to post some of the JSON file so I can see the structure.

**Reply by ebrummer** [reply]
Thanks. I think I'll attempt doing that.

**Reply by ebrummer** [reply]
Is this the best way of scripting this?
FOR /F "usebackq tokens=*" %%A in (C:\postlist.txt) do py -m bdfr clone C:\SaveLocation --link %%A --comment-context --verbose

**Reply by Serene-Arc** [reply]
If you have a list of IDs, we have the option to include an ID file naturally which will be better than this since the BDFR doesn't have to restart completely. See the README for details.
